package exception.trycatch;

public class MainClass {
	public static void main(String[] args) {
		//실행 예외 종류
		//nullpointer
//		String str = null;
//		str.charAt(1);
//		
//		//ArrayIndexBoundsExcpeption
//		int[] arr = {1,2,3};
//		arr[10] = 10;

		//NumberFormat
		int a = 1;
		System.out.println(a+"");
		String s= "1";
		Integer.parseInt(s);
		System.out.println(s);
		
	}
}
